geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/delta/RemoveByteArrays.java

```diff
@@ -53,14 +51,14 @@ public void add(byte[] delta) {
 
   public void serializeTo(DataOutput out) throws IOException {
     super.serializeTo(out);
-    InternalDataSerializer.writeArrayLength(byteArrays.size(), out);
+    DataSerializer.writePrimitiveInt(byteArrays.size(), out);
```

Comment:
Sadly, this is going to make this array size take more bytes on the wire - writeArrayLength did some compression on on the int, but writePrimativeInt does not. So maybe we should still use the internal class? Other option would be `InternalDataSerializer.writeUnsignedVL` - which also compresses the integer.